src/lavinmq/replication/server.cr

```diff
@@ -173,8 +188,7 @@ module LavinMQ
 
       class Follower
         Log = ::Log.for(self)
-
-        getter ack = Channel(Nil).new
+        getter ack = Channel(Int64).new
```

Comment:
I don't think this getter is used, it seem we always access it via the instance variable.

```suggestion
        @ack = Channel(Int64).new
```

Reply:
you're right!